docs/release_process.md

```diff
@@ -40,8 +40,18 @@ Let's make sure there are no migration conflicts with the previous release:
     $ git checkout master
     $ make migrate
```

Comment:
The person making the 0.22.0 release will follow the documentation as it is written at the time of the release.

Therefore, the first part of the instructions (the current ones) are kept so that we can make the 0.22.0 release. :)

As I said, I'm planning to remove them as soon as the 0.22.0 release is out.